hack/jenkins/minikube_cross_build_and_upload.sh

```diff
@@ -29,10 +29,15 @@ export GOPATH=/var/lib/jenkins/go
 
 docker kill $(docker ps -q) || true
 docker rm $(docker ps -aq) || true
-
+set +e
 make -j 16 all
+set -e
+
 gsutil cp gs://minikube-builds/logs/index.html gs://minikube-builds/logs/${ghprbPullId}/index.html
 
+# Exit if the cross build failed.
+if [ "$?"-ne 0]; then echo "cross build failed"; exit 1; fi
```

Comment:
Should this be above the gsutil cp?

Reply:
No, because then we'll exit before we upload the logs.  We want the logs regardless.  Right now `make all` will fail and exit without copying the logs over

Reply:
Ah, right. LGTM.